### PR TITLE
Add dateRecorded column to Cancer Disease Status

### PR DESCRIPTION
**Pull request by jafeltra**
I realized that the `dateRecorded` column was removed from the CancerDiseaseStatus CSV sheet. We had added this column to reflect the date the entry was added to the CSV sheet and use it to filter out data during extraction. However, since this doesn't map onto an element in the FHIR resource, it isn't reflected in Rob's spreadsheet. I can update the spreadsheet as well, as long as everyone agrees we should still have this column.

### CODE REVIEW

**Review by kjmahalingam — APPROVED** [review]
💯 🥇

**Review by Dtphelan1 — APPROVED** [review]

**Review by kjmahalingam — APPROVED** [review]
I was there when these walls were built, and I couldn’t be more proud to call this place home.

File: src/cli.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ program
   .usage('[options]')
   .option('-f --from-date <date>', 'The earliest date and time to search')
   .option('-t --to-date <date>', 'The latest date and time to search')
-  .option('-a, --all-entries', 'Flag to indicate not to filter data by date')
+  .option('-a, --all-entries', 'Flag to indicate not to filter data by date', true)
+  .option('--no-all-entries', 'Flag to indicate not to filter data by date')
```
Comment (Dtphelan1) [inline src/cli.js:14]: DP1 - Need to change the help text here

Comment (jafeltra) [reply src/cli.js:14]: Ah my bad

Comment (jafeltra) [reply src/cli.js]: Committed and force pushed (shh)

**Review by julianxcarter — APPROVED** [review]
Just tested and this checks out. Nice work guys!

**Pull request merged**